homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -281,27 +282,27 @@ async def _try_command(self, mask_error, func, *args, **kwargs):
             _LOGGER.error(mask_error, exc)
             return False
 
-    async def async_start(self):
+    async def async_start(self) -> None:
         """Start or resume the cleaning task."""
         await self._try_command(
             "Unable to start the vacuum: %s", self._device.resume_or_start
         )
 
-    async def async_pause(self):
+    async def async_pause(self) -> None:
         """Pause the cleaning task."""
         await self._try_command("Unable to set start/pause: %s", self._device.pause)
 
-    async def async_stop(self, **kwargs):
+    async def async_stop(self, **kwargs: Any) -> None:
         """Stop the vacuum cleaner."""
         await self._try_command("Unable to stop: %s", self._device.stop)
 
-    async def async_set_fan_speed(self, fan_speed, **kwargs):
+    async def async_set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
         """Set fan speed."""
         if fan_speed in self.coordinator.data.fan_speeds:
-            fan_speed = self.coordinator.data.fan_speeds[fan_speed]
+            fan_speed_int = self.coordinator.data.fan_speeds[fan_speed]
```

Comment:
Small code change here, as mypy does like having a new type (`int`) for `fan_speed`.